bundles/AdminBundle/src/Controller/Admin/Document/DocumentControllerBase.php

```diff
@@ -289,14 +290,6 @@ protected function getPropertyInheritance(Model\Document $document, string $prop
         return false;
     }
 
-    /**
-     * @template T of Model\Document\PageSnippet
-     *
-     * @param T $document
-     * @param null|Version $draftVersion
-     *
-     * @return T
```

Comment:
@mcop1 Why do you remove this? This say that the type of the $document param is the same as the return type

Reply:
Good catch, thanks. Some of the php docs, which are using templates, were marked as unnecessary, I didn´t remove most of them, but must have missed that one.